lib/cocina/models/builders/title_builder.rb

```diff
@@ -24,24 +24,46 @@ def self.build(titles, strategy: :first, add_punctuation: true)
           new(strategy: strategy, add_punctuation: add_punctuation).build(titles)
         end
 
+        # the "main title" is the title withOUT subtitle, part name, etc.  We want to index it separately so
+        #   we can boost matches on it in search results (boost matching this string higher than matching full title string)
+        #   e.g. "The Hobbit" (main_title) vs "The Hobbit, or, There and Back Again (full_title)
+        # @param [[Array<Cocina::Models::Title,Cocina::Models::DescriptiveValue>] titles the titles to consider
+        # @param [Symbol] strategy ":first" is the strategy for selection when primary or display
+        #   title are missing
+        # @param [Boolean] add_punctuation determines if the title should be formmated with punctuation
+        # @return [String] the title value for Solr
+        def self.main_title(titles)
+          # 245a = title.structuredValue.value with type 'main title' (or just title.value),
+          # 245b = title.structuredValue.value with type 'subtitle'; if there are multiple titles, the 245 should have status 'primary' or at least be first
+          new(strategy: :first, add_punctuation: false).main_title(titles)
+        end
+
         def initialize(strategy:, add_punctuation:)
           @strategy = strategy
           @add_punctuation = add_punctuation
         end
 
-        # @param [[Array<Cocina::Models::Title>] titles the titles to consider
+        # @param [[Array<Cocina::Models::Title>] cocina_titles the titles to consider
         # @return [String] the title value for Solr
-        def build(titles)
-          cocina_title = primary_title(titles) || untyped_title(titles)
-          cocina_title = other_title(titles) if cocina_title.blank?
+        def build(cocina_titles)
+          cocina_title = primary_title(cocina_titles) || untyped_title(cocina_titles)
+          cocina_title = other_title(cocina_titles) if cocina_title.blank?
 
           if strategy == :first
             extract_title(cocina_title)
           else
-            cocina_title.map { |one| extract_title(one) }
+            cocina_titles.map { |one| extract_title(one) }
```

Comment:
Can you say more about this change? I think I grok the variable name changes in this method, but this (naïvely) appears to be a functional change.

Reply:
This method is broken, and this is one way.  Why would you call map on a single title object?

I did the rename for my own sanity and ... voila - it became clear that this should be the plural.

Reply:
plus ... specs.